File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    count = 0
    for i in range(len(t)-len(p) +1) :
        # print(i)
        part_t = t[i:(i+len(p))]
        if (int(part_t) - int(p) <= 0) :
            count += 1
    return count
    
        
    # answer = 0
    # return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let cnt = 0;
    for (let i=0; i<t.length-p.length+1; i++) {
        if(Number(p)>=Number(t.substr(i,p.length))) {
            cnt+=1
        }
    }
    return cnt

    // var answer = 0;
    // return answer;
}